docs/en/observability/synthetics-journeys.asciidoc

```diff
@@ -12,6 +12,9 @@ Start by learning the basics of synthetic monitoring, including how to:
 
 * <<synthetics-create-test>>
 * <<synthetic-run-tests>>
+* <<synthetics-manage-monitors>>
+
+image::images/synthetic-monitor-lifecycle.png[Diagram of the lifecycle of a synthetic monitor: write a test, test it locally, create a monitor, manage a monitor, delete a monitor]
```

Comment:
Added the diagram I shared, but I can remove if we think it emphasizes working locally too much.